Sources/SPMBuildCore/BuildParameters/BuildParameters+Testing.swift

```diff
@@ -75,9 +72,9 @@ extension BuildParameters {
             switch self {
             case .loadableBundle:
                 try container.encode(DiscriminatorKeys.loadableBundle, forKey: ._case)
-            case .entryPointExecutable(let explicitlyEnabledDiscovery, let explicitlySpecifiedPath):
+            case .entryPointExecutable(let explicitlySpecifiedPath):
                 try container.encode(DiscriminatorKeys.entryPointExecutable, forKey: ._case)
-                try container.encode(explicitlyEnabledDiscovery, forKey: .explicitlyEnabledDiscovery)
+                try container.encode(false, forKey: .explicitlyEnabledDiscovery)
```

Comment:
A comment here explaining why this remains in place might be helpful for future maintainers.

Reply:
I think this will be gone in a future commit in this same PR, we don't expect build parameters serialization to be compatible between versions.

Reply:
Okay! I'm fine with that.